Homework - first page

SergeyLavrentyev opened the pull request:
сделал разметку первой страницы

Commit:
Homework - first page 

сделал разметку первой страницы

tahavn reviewed (changes requested):
Правки ДЗ2

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+  <body>
+    <header>
+      <a class="header-logo" href="#"><img src="/logo.png" alt="logo"/></a>
+      <nav class="header-logo">
```
странный класс для навигации

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+        <li class="slider-line"></li>
+        <li class="slider-line"></li>
+      </ul>
+      <h1>THE OOH NOO<br />COLLECTION</h1>
```
1. как можно меньше используй тег br, в данном случае можно стилями ограничить ширину блока из-за чего текст перенесется. 
2. Это заголовок слайда, по макету видно, что их будет несколько и поэтому тег h1 не подходит, т.к. он может быть один на странице

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+      <h1>THE OOH NOO<br />COLLECTION</h1>
+      <p>Designed in Slovenia</p>
+      <button class="button button_black">Explore Now</button>
+      <p>Free (continental) USA shipping on all orders of $100 USD or more!</p>
```
не хватает класса, через строку выше тоже тег p, как будешь стилизовать?

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+        Modern finds for the modern home. Curated goods by unique designers
+        around the world from fashion to furniture for kids and adults.
+      </p>
+    </section>
```
не хватает классов, как будешь стилизовать?

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+        <li><h3>APPAREL KIDS</h3></li>
+        <li><h3>HOLIDAY SHOP</h3></li>
+        <li><h3>TOYS</h3></li>
+        <li><h3>ACCESSORIES</h3></li>
```
здесь должны быть ссылки, т.к. пользователи заходят посмотреть какие товары есть в категории

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+      <ul>
+        <li>Delivery</li>
+        <li>Guarantee</li>
+        <li>Credit</li>
```
по дизайну видно, что эти элементы должны быть кликабельные. Требуется уточнить функционал, если это табы, то должны быть button, если ссылки на другие страницы, то соответственно ссылки

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+        beautiful designers that we work with to thank!
+      </p>
+      <a href="" class="button button_black"></a>
+    </section>
```
не хватает классов, как будешь стилизовать?

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+    </section>
+    <article class="about">
+      <h2>Design Life Kids</h2>
+      <h3>About Us</h3>
```
Заголовок ставится на какой-то логический блок. Вместо h3, в данном случае, лучше использовать div

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+        by. It's simply magical.
+      </p>
+      <img src="/map.png" alt="map" />
+    </article>
```
не хватает классов, как будешь стилизовать?

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+      <p>Company Rating</p>
+      <p>Copyright © 2018 Dlk • Design Life Kids</p>
+      <p>Web Design — Direct Line Development</p>
+    </footer>
```
не хватает классов, как будешь стилизовать?

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+      <h2>Connect</h2>
+      <form>
+        <input type="email" name="email" placeholder="Email" />
+        <button>Join</button>
```
для форм у кнопки лучше указывать тип, не все типы подходят для отправки форм

tahavn commented on index.html:
```diff
@@ -0,0 +1,142 @@
+      </ul>
+      <p>Company Rating</p>
+      <p>Copyright © 2018 Dlk • Design Life Kids</p>
+      <p>Web Design — Direct Line Development</p>
```
не хватает общей обертки, которая потом облегчит стилизацию

tahavn reviewed (changes requested):
Правки ДЗ2

Commit:
Правки ДЗ

добавил классы, кнопки, тип для кнопки, обертка для элементов в футере

Commit:
homework update

классы в  футер

tahavn commented on index.html:
```diff
@@ -0,0 +1,149 @@
+        <li class="slider-line active"></li>
+        <li class="slider-line"></li>
+        <li class="slider-line"></li>
+        <li class="slider-line"></li>
```
если предполагается, что это навигация слайдера, то лучше использовать button

SergeyLavrentyev replied:
то есть список убрать?

tahavn replied:
список можно оставить, но внутрь li добавить button

tahavn commented on index.html:
```diff
@@ -0,0 +1,149 @@
+      </ul>
+      <h2>THE OOH NOO COLLECTION</h2>
+      <p class="slider-subtitle">Designed in Slovenia</p>
+      <button class="button button_black">Explore Now</button>
```
рекламные баннеры обычно куда-то ведут. надо использовать ссылку.

SergeyLavrentyev replied:
а модальное окно из макета не относится к кнопке?

tahavn replied:
к кнопке, но есть страница каталога, и ссылка будет на нее

tahavn replied:
к кнопке, но есть страница каталога, и ссылка будет на нее

tahavn commented on index.html:
```diff
@@ -0,0 +1,149 @@
+      <p class="category-subtitle">
+        Modern finds for the modern home. Curated goods by unique designers
+        around the world from fashion to furniture for kids and adults.
+      </p>
```
1.По дизайну здесь еще должны быть кнопки.
2. на мой взгляд не очень подходящий класс, лучше category-description или просто category-text

subtitle рассматривать в рамках этого макета, то лучше всего такой класс смотрелся в блоке Design Life Kids на тексте About Us

tahavn commented on index.html:
```diff
@@ -0,0 +1,149 @@
+        started as an outlet to deal with the traumatic loss of my husband, dlk
+        has now become a truly happy place for me and i have you and all of the
+        beautiful designers that we work with to thank!
+      </p>
```
В рамках учебного проекта твоя разметка этого блока подойдет.
Но лучше задать родительский div ему дать класс, а внутрь положить два p без классов. Если смотреть сильно вперед то с твоей версткой будут еще работать бэкэндеры и если использовать твою структуру то в админ-панели придется добавлять два отдельных поля чтобы вывести так текст.

tahavn commented on index.html:
```diff
@@ -0,0 +1,149 @@
+        <p class="rating">Company Rating</p>
+        <p class="copyright">Copyright © 2018 Dlk • Design Life Kids</p>
+        <p class="design-company">Web Design — Direct Line Development</p>
+      </div>
```
все h2 выглядят одинаково, есть смысл им задать один какой-то класс, чтобы не прописывать отдельно для каждого одинаковые стили. 

и для меню, можно оставить те классы которые есть и добавить еще по одному одинаковому для всех, в котором задать цвет, размер шрифта, отступы. Хотя навигация очень похожа можно было обойтись и одними классами для всех меню подвала

tahavn commented on index.html:
```diff
@@ -0,0 +1,149 @@
+        <li class="social-list-item"><a href="#">Instagram</a></li>
+      </ul>
+      <div class="company-info">
+        <p class="rating">Company Rating</p>
```
не хватает звезд рейтинга, можно вставить картинкой, а лучше нагуглить как верстаются звезды рейтинга, это очень легко

SergeyLavrentyev replied:
подключть font awsome?

tahavn replied:
зачем? можно использовать изображения или спецсимволы

tahavn replied:
поэтому я и писал погуглить. есть несколько вариантов как решить подобную задачу. Повторюсь, можешь вставить картинкой.

tahavn reviewed (changes requested):
Правки ДЗ2

tahavn reviewed (changes requested):
Правки ДЗ2

Commit:
Update index.html

добавил  кнопки в слайдер, рейтинг компании,  ссылку в баннер, обернул текст в div, добавил классы в футер

tahavn commented on index.html:
```diff
@@ -0,0 +1,176 @@
+      </ul>
+      <h2>THE OOH NOO COLLECTION</h2>
+      <p class="slider-subtitle">Designed in Slovenia</p>
+      <a class="button button_black">Explore Now</a>
```
Вот здесь надо будет добавить дополнительную обертку, может даже две, div к примеру. 
Но это не критично, как начнешь стилизовать, станет ясно для чего обертки.

tahavn commented on index.html:
```diff
@@ -0,0 +1,176 @@
+      <p class="category-text">
+        Modern finds for the modern home. Curated goods by unique designers
+        around the world from fashion to furniture for kids and adults.
+      </p>
```
для стилизации понадобиться дополнительная обертка для этого блока, к примеру div

tahavn commented on index.html:
```diff
@@ -0,0 +1,176 @@
+          <span class="star">&#9733;</span>
+        </div>
+        <p class="copyright">Copyright © 2018 Dlk • Design Life Kids</p>
+        <p class="design-company">Web Design — Direct Line Development</p>
```
Ссылку на веб-студию, лучше размещать в ссылке, для рекламы полезно

tahavn reviewed (changes requested):
Рекомендации ДЗ2

Commit:
Update index.html

обертки для контента слайдера и стрелок  навигации.  веб студия ссылкой.

tahavn reviewed (commented):
ДЗ2
